scripts/install-docker.sh

```diff
@@ -4,7 +4,4 @@
 export DEBIAN_FRONTEND=noninteractive
 export TZ=America/Los_Angeles
 
-apt-get update -qq && apt-get upgrade -qq
```

Comment:
In `.buildkite/test-rayjob-sample-yamls.yml`, a `curl` command is executed before `install-docker.sh`, so there's no need to install curl again.